src/x86-64.h

```diff
@@ -412,11 +345,9 @@
  * some newer CPUs capable of SSE4.2 but not AVX happen to lack SMT, so will
  * likely benefit from the 3x interleaving with no adverse effects for the
  * multi-threaded case.
- *
- * In Jumbo, we may get BF_X2 from autoconf (after testing ht cpuid flag).
```

Comment:
So we decided to no longer check HT flag in configure? I'm OK with that.

Reply:
I recall some discussion of that flag being unreliable at best

Reply:
Yes, the HT flag is "unreliable at best". I am also considering just making AVX+ builds use BF_X3 as well, based on my latest benchmarks on newer systems and with newer gcc - there are regressions in newer gcc for the BX_X2=1 case, which BF_X3 avoids. %-)